shared/src/androidTest/kotlin/co/touchlab/kampkit/TestAnnotations.kt

```diff
@@ -3,3 +3,10 @@ package co.touchlab.kampkit
 actual typealias RunWith = org.junit.runner.RunWith
 actual typealias Runner = org.junit.runner.Runner
 actual typealias AndroidJUnit4 = androidx.test.ext.junit.runners.AndroidJUnit4
+
+// Our expect declaration has fewer parameters than the actual Config annotation
+// If we try to match the expect declaration exactly, we run into problems because the `application` parameter has
+// a default value `DefaultApplication::class` which we can't write an actual declaration for because it's
+// package-private
+@Suppress("NO_ACTUAL_CLASS_MEMBER_FOR_EXPECTED_CLASS")
+actual typealias Config = org.robolectric.annotation.Config
```

Comment:
This got messier than I thought at first glance so I did this hack instead.

Increasingly feeling like maybe better to just remove Robolectric since we don't actually need it for our usage but I still kind of like having a demo of it in case anyone does need it. Anyone else have opinions?

Reply:
Having some example of `Roboelectric` would be helpful to others as it's still very popular, and a lot of people might be using it when they're trying to switch to KMM

Reply:
I'd advocate for removing robolectric if we don't need it. Feel like having workarounds without clear usage would add to confusion. Particularly if they aren't keeping up with the versions. If we want to create a demo for it, we should identify use cases where it's actually needed

Reply:
Sounds good to me @samhill303